ga4gh/cli.py

```diff
@@ -364,7 +388,10 @@ def client_main(parser=None):
         "--workarounds", "-w", default='', help="The workarounds to use")
     parser.add_argument(
         "--key", "-k", help="The auth key to use")
-    subparsers = parser.add_subparsers(title='subcommands',)
+    parser.add_argument(
+        "--outputFormat", "-o", help="The output format to use",
+        default="minimal", choices=["minimal", "extended"])
```

Comment:
A boolean flag would seem more natural here. What about -O, --minimalOutput, default=False? We keep the semantics we  have from before then and we can just add the --minimalOutput flag for your testing script.